package.json

```diff
@@ -291,6 +301,11 @@
         "title": "ONE: Set Default Values",
         "category": "ONE"
       },
+      {
+        "command": "one.editor.cfg.setDefaultEdgetpuValues",
```

Comment:
```suggestion
        "command": "one.editor.edgetpucfg.setDefaultValues",
```

Reply:
Could you use this? I think it's better to distinguish each editor panels : `cfg editor` and `edge tpu cfg editor` because they use different resource javscript/css files.

Reply:
Other things looks okay. Thanks.

Reply:
> Could you use this? I think it's better to distinguish each editor panels : `cfg editor` and `edge tpu cfg editor` because they use different resource javscript/css files.

I changed the command name!
Thanks for your review!